--- vm-memory-0.12.2/Cargo.toml	1970-01-01T00:00:01+00:00
+++ vm-memory-0.12.2/Cargo.toml	2023-08-30T13:37:32.818256+00:00
@@ -21,6 +21,7 @@
 categories = ["memory-management"]
 license = "Apache-2.0 OR BSD-3-Clause"
 repository = "https://github.com/rust-vmm/vm-memory"
+exclude = ["rust-vmm-ci/*", "/coverage_config_*", "benches/*"]
 
 [package.metadata.docs.rs]
 all-features = true
@@ -51,9 +52,6 @@
 version = "0.11.0"
 optional = true
 
-[dev-dependencies.criterion]
-version = "0.3.0"
-
 [dev-dependencies.matches]
 version = "0.1.0"
 
