diff --git a/Cargo.toml b/Cargo.toml
index c0007bd..d23a46e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,6 +22,7 @@ categories = ["memory-management"]
 license = "Apache-2.0 OR BSD-3-Clause"
 repository = "https://github.com/rust-vmm/vm-memory"
 resolver = "2"
+exclude = ["rust-vmm-ci/*", "/coverage_config_*", "benches/*"]
 
 [package.metadata.docs.rs]
 all-features = true
@@ -41,9 +42,6 @@ optional = true
 [dependencies.libc]
 version = "0.2.39"
 
-[dev-dependencies.criterion]
-version = "0.3.0"
-
 [dev-dependencies.matches]
 version = "0.1.0"
 
