diff --git a/Cargo.toml b/Cargo.toml
index 6590ac2..da935b4 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,6 +22,7 @@ keywords = ["memory"]
 categories = ["memory-management"]
 license = "Apache-2.0 OR BSD-3-Clause"
 repository = "https://github.com/rust-vmm/vm-memory"
+exclude = ["rust-vmm-ci/*", "/coverage_config_*", "benches/*"]
 [profile.bench]
 lto = true
 codegen-units = 1
@@ -49,6 +50,3 @@ backend-atomic = ["arc-swap"]
 backend-bitmap = []
 backend-mmap = []
 default = []
-[target."cfg(windows)".dependencies.winapi]
-version = ">=0.3"
-features = ["errhandlingapi", "sysinfoapi"]
